hat/assets/js/apps/Iaso/domains/forms/index.tsx

```diff
@@ -1,79 +1,42 @@
 import React, { FunctionComponent } from 'react';
-import { Box, Grid } from '@mui/material';
+import { Box } from '@mui/material';
 import { makeStyles } from '@mui/styles';
 import { useSafeIntl, commonStyles } from 'bluesquare-components';
-import DownloadButtonsComponent from '../../components/DownloadButtonsComponent';
 import TopBar from '../../components/nav/TopBarComponent';
 import { baseUrls } from '../../constants/urls';
-import { useQueryString } from '../../hooks/useApiParams';
 import { useParamsObject } from '../../routing/hooks/useParamsObject';
 import { Filters } from './components/Filters';
 import { FormsTable } from './components/FormsTable';
-import { tableDefaults, useGetForms } from './hooks/useGetForms';
+import { useGetForms } from './hooks/useGetForms';
 import MESSAGES from './messages';
+import { FormsParams } from './types/forms';
 
 const useStyles = makeStyles(theme => ({ ...commonStyles(theme) }));
 
-const dwnldBaseUrl = '/api/forms';
-
-export type Params = {
-    pageSize: string;
-    order: string;
-    page: string;
-    search?: string;
-    showDeleted?: string;
-    planning?: string;
-    projectsIds?: string;
-    isSearchActive?: string;
-    showInstancesCount?: string;
-};
-
 const Forms: FunctionComponent = () => {
     const baseUrl = baseUrls.forms;
-    const params = useParamsObject(baseUrl) as unknown as Params;
+    const params = useParamsObject(baseUrl) as unknown as FormsParams;
     const classes = useStyles();
     const { formatMessage } = useSafeIntl();
 
-    const isSearchActive = params?.isSearchActive === 'true';
     // This doesn't duplicate API calls as long as
     // the querystring generated by useGetForms is the same as the one in the table
     const { data: forms, isLoading: isLoadingForms } = useGetForms(
         params,
         undefined,
-        isSearchActive,
+        true,
```

Comment:
This makes me think it would be more legible if `useGetForms` took a dictionary as param. 
`useGetForms(params, undefined, true)` is impossible to understand without having to check the hook's code